### DISCUSSION TITLE
goodbyedpi-0.2.3rc1 (не могу удалить WinDivert64.sys)

### DISCUSSION THREAD

**Discussion by WinDivert64sys** [community/community#134676] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

хотел перенести папку с программой, но WinDivert64.sys остался на прежнем месте, его даже удалить нельзя, потому что пишет, что он используется \ открыт , его нельзя найти через диспетчер задач что бы завершить процесс, его вообще не понятно где можно найти что бы закрыть, его даже Unlocker не удаляет
что делать?

**Comment by Vitobeyondly** [comment]
через анлокер и удаляй, он просто при следующем запуске системы его сотрет. Либо через безопасный режим.

**Comment by ebndev** [comment]
Hi @WinDivert64sys, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
